solidity/random-beacon/contracts/DKGValidator.sol

```diff
@@ -209,16 +209,23 @@ contract DKGValidator {
             )
         );
 
-        address[] memory membersAddresses = sortitionPool.getIDOperators(
-            result.members
+        uint32[] memory signingMemberIds = new uint32[](
+            result.signingMembersIndices.length
+        );
+        for (uint256 i = 0; i < result.signingMembersIndices.length; i++) {
+            signingMemberIds[i] = result.members[
+                result.signingMembersIndices[i] - 1
+            ];
+        }
+
+        address[] memory signersAddresses = sortitionPool.getIDOperators(
```

Comment:
signersAddresses -> signingMemberAddresses? Same comment for line 237

Reply:
Done in https://github.com/keep-network/keep-core/pull/2745/commits/000b4bce8772a6e22211166aaf9b7fef0f91dcd1.